src/components/DashboardPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Card, Button, Container, Row, Col} from 'react-bootstrap';
+
+class Dashboard extends React.Component {
+  render() {
+    return (
+      <container>
```

Comment:
Should be "Container" with a capital C. Otherwise this causes an error in the browser.

Reply:
fixed